src/views/CreateView.vue

```diff
@@ -86,12 +87,12 @@
                           v-for="exercise in exercisesToShow"
                           :key="exercise.name"
                           class="exercise-to-select"
-                          :class="{'mt-8': selectedExercise !== ''}"
+                          :class="{'mt-8': selectedExercise !== null}"
                         >
                           
                           <img
-                            v-show="selectedExercise !== ''"
-                            @click="selectedExercise = ''"
+                            v-show="selectedExercise !== null"
```

Comment:
Same https://github.com/Kepmon/Workout-planner/pull/11/files#r1151510745

Reply:
**I guess it's corrected**